pkg/plugin/processor/builtin/impl/avro/decode.go

```diff
@@ -108,7 +108,7 @@ func NewDecodeProcessor(logger log.CtxLogger) sdk.Processor {
 func (p *decodeProcessor) Specification() (sdk.Specification, error) {
 	return sdk.Specification{
 		Name:    "avro.decode",
-		Summary: "Decodes a field's raw data in the Avro format",
+		Summary: "Decodes a field's raw data in the Avro format.",
```

Comment:
Nitpick: didn't we say summaries should NOT end in dots, since they're shown as headers?

Reply:
That's the case for example summaries, not specifications, these are shown below the header 🙈